services/subnet_manager/src/main/java/com/futurewei/alcor/subnet/entity/IPState.java

```diff
@@ -0,0 +1,51 @@
+package com.futurewei.alcor.subnet.entity;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.futurewei.alcor.common.entity.CustomerResource;
+
+public class IPState  extends CustomerResource {
```

Comment:
We could update the ip related data structure to be aligned with the IP manager PR.

Reply:
Okay ~ I'll update it